app/assets/javascripts/geocoding.js

```diff
@@ -0,0 +1,30 @@
+$(document).ready(function() {
+    var error, getReverseGeocodingData, success;
+    error = function(msg) {
+        document.getElementById('issue_location').value = msg + ' Введіть адресу проблеми вручну';
+    };
+    success = function(position) {
+        document.getElementById('issue_longitude').value = position.coords.longitude;
+        document.getElementById('issue_latitude').value = position.coords.latitude;
+        getReverseGeocodingData(position.coords.latitude, position.coords.longitude);
+    };
+    getReverseGeocodingData = function(lat, lng) {
+        var geocoder, latlng;
+        latlng = new google.maps.LatLng(lat, lng);
+        geocoder = new google.maps.Geocoder;
+        geocoder.geocode({
+            'latLng': latlng
+        }, function(results, status) {
+            var location;
+            if (status === google.maps.GeocoderStatus.OK) {
+                location = results[0].formatted_address;
+                document.getElementById('issue_location').value = location;
+            }
+        });
+    };
+    if (navigator.geolocation) {
+        navigator.geolocation.getCurrentPosition(success, error);
+    } else {
+        error('Геолокація не підтримується вашим браузером');
```

Comment:
do not put this message, it will disturb users (if it works (geolocation) - ok, if not - ok too)